docs/platform/release-notes/tizen-5-5-m2.md

```diff
@@ -0,0 +1,641 @@
+  - Xamarin sample applications to provide how to make custom pipeline with Tizen gstreamer plugins have been added.
+  - Simple guideline for Tizen Xamarin application has been added.
+- Media Player
+  - API set for audio offloading has been added.
+  - API to query about supported format for push media stream has been added.
+  - Support for audio pcm extraction has been added.
+  - Support for setting audio codec type has been added.
+- Vision
+  - Deep Neural Network (DNN) based high level vision inference API set, such as image classification, object detection, face and its landmark detection, have been added.
+    - tensorflow-lite model data has been supported.
+    - caffe and tensorflow model data with OpenCV has been supported.
+- Media Controller
+  - API set to support subtitle, 360 mode, display mode, display rotation have been added.
+  - API set for ability has been added.
+  - New error for ability not supported has been added.
+- Image Utility
+  - API set for decoding and encoding image has been updated.
+- C# API
+  - AudioIO
+    - API set to support multi channel and signed 32-bit audio sample have been added.
+  - AudioManager
+    - Audio ducking API set has been added.
+    - API to support network device has been added.
+    - API to get preferred input, output device has been added.
+  - ImageUtil
+    - API set to extract thumbnail synchronously have been added.
+    - The condition of resizing thumbnail output size has been changed.
+  - MediaCodec
+    - The feature and exception tag of EventArgs class has been removed.
+  - MediaContent
+    - API to add face information has been added.
+  - MediaController
+    - API set to create instance without bundle have been added.
+    - API set to support subtitle, 360 mode, display mode, display rotation have been added.
+  - MediaTool
+    - API set to support PCM audio channel mapping have been added.
+  - MediaPlayer
+    - API set to export decoded audio, video data have been added.
+    - The cancelable version of PrepareAsync API has been added.
+    - API to get supported audio and video types has been changed. The API can get all supported codec types defined by each target.
+    - API to use audio offload has been added.
+  - MediaVision
+  - API set to support inference face detection, facial landmark detection, object detection, image classification have been added.
+- Multimedia Daemon
+  - Resource-manager
+    - New resource for audio offload has been added.
+
+### Fixes
+
+- C# API
+  - AudioIO
+    - Crash issue caused by using GC collected delegate has been fixed.
+    - Block and crash issue caused by thread deadlock have been fixed.
+  - AudioManager
+    - Crash issue caused by marshaling safe handle has been fixed.
+  - Camera
+    - Missed exception of constructor has been fixed.
+  - ImageUtil
+    - Crash issue caused by marshaling buffer has been fixed.
+    - Deprecated native p/invoke function has been changed.
```

Comment:
- Native p/invoke function has been deprecated.

Reply:
It means that native (p/invoke) functions called by C# ImageUtil API are deprecated, so I changed it to newly added native functions. 
I've thought about this sentence again, and I think it can be removed.
Please simply remove this sentence.